readthedocs/rtd_tests/tests/test_api.py

```diff
@@ -152,6 +152,44 @@ def test_make_build_commands(self):
         self.assertEqual(build['commands'][0]['run_time'], 5)
         self.assertEqual(build['commands'][0]['description'], 'foo')
 
+    def test_get_raw_log(self):
+        build = get(Build, project_id=1, version_id=1, builder='foo')
+        get(
+            BuildCommandResult,
+            build=build,
+            command='python setup.py install',
+            output='Installing dependencies...'
+        )
+        get(
+            BuildCommandResult,
+            build=build,
+            command='git checkout master',
+            output='Switched to branch "master"'
+        )
+        client = APIClient()
+
+        api_user = get(User, user='test', password='test')
+        client.force_authenticate(user=api_user)
+        resp = client.get('/api/v2/build/{0}/log.txt'.format(build.pk))
+        self.assertEqual(resp.status_code, 200)
+        resp = client.get(
+            '/api/v2/build/{0}/log/'.format(build.pk),
+            format='txt'
+        )
+        self.assertEqual(resp.status_code, 200)
+
+        self.assertIn('RTD build information start', resp.data)
+        self.assertIn('RTD build information end', resp.data)
+        self.assertIn('[rtd-command-info]', resp.data)
+        self.assertIn(
+            'python setup.py install\nInstalling dependencies...',
+            resp.data
+        )
+        self.assertIn(
+            'git checkout master\nSwitched to branch "master"',
+            resp.data
+        )
```

Comment:
Let me know if you want me to make a more _robust_ test for checking the response.

Reply:
You can also add something like:

    self.assertEqual(resp.data, build.raw_log)